### DISCUSSION TITLE
Student plan activation

### DISCUSSION THREAD

**Discussion by joelmohh** [community/community#66615] [GitHub Education] [answered]
### Hi everyone,

How long does it take for a Github student license to take effect after activation?

**Comment by arafajs** [comment, marked as answer]
It usually takes 72 hours for your GitHub Student Developer Pack benefits to take effect after activation. However, during busy periods, such as the start of a new semester, it may take a bit longer.

**Comment by nagasaikushal** [comment]
![image](https://github.com/community/community/assets/117629127/6f88abb9-28ef-4c11-bdc1-343ce0d63d68)
I applied for student benefits but it didn't apply in time and I cant use this . Because of this I lost the chance of appling for github campus experts.

**Comment by rb-fox** [comment]
Will it activate in visual studio automatically when they get to it?
![image](https://github.com/community/community/assets/116045284/b2dac7b9-baaf-4c69-8232-2db2d60467ff)
